src/styles/bootstrap-theme/colors.module.scss

```diff
@@ -27,18 +27,31 @@ $dark: $slate;
 $light: $platinum;
 $info: $sea-green;
 
-$custom-colors: (
-  'white': $white,
+$theme-colors: (
+  'primary': $primary,
+  'secondary': $secondary,
+  'success': $success,
+  'info': $info,
+  // 'warning': $meyer-lemon,
+  'danger': $danger,
+  'light': $light,
   'loquat': $loquat,
-  'meyer-lemon': $meyer-lemon,
-  'medium-gray': $medium-gray,
+  'mediumGray': $medium-gray,
+  'dark': $dark,
+  'white': $white,
   'rating-1': $cherry,
```

Comment:
Convention has the keys with camelCase (hence the `mediumGray`). Should anything be done about the 'rating' names (e.g. `ratingOne` instead of `rating-1`), or is it fine as it is?

Reply:
These should remain in kebab-case to follow css/classname conventions. Declaring them in camelcase means that bootstrap will create utilities like `text-mediumGray`, `text-meyerLemon`, which would break all of our existing css classes.